shell/platform/darwin/macos/framework/Source/FlutterViewController.mm

```diff
@@ -360,8 +360,8 @@ - (void)listenForMetaModifiedKeyUpEvents {
 
 - (void)configureTrackingArea {
   if (_mouseTrackingMode != FlutterMouseTrackingModeNone && self.view) {
-    NSTrackingAreaOptions options =
-        NSTrackingMouseEnteredAndExited | NSTrackingMouseMoved | NSTrackingInVisibleRect;
+    NSTrackingAreaOptions options = NSTrackingMouseEnteredAndExited | NSTrackingMouseMoved |
+                                    NSTrackingInVisibleRect | NSTrackingEnabledDuringMouseDrag;
```

Comment:
Interesting, I didn't know about this. My last deep wrestling with complex mouse tracking predated 10.5 IIRC :)

Reply:
Yeah. The default behavior where you get mouse exit but not enter during the same drag is just weird.